bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -138,12 +138,18 @@ JsFunction
 {
     {% if type.custom_from_rs_bytes -%}
     let custom_from_rs_bytes = {{ type.custom_from_rs_bytes }};
+    // TODO: Some references don't need mutability
```

Comment:
I wouldn't put those comments as TODO: fixing those will only make clippy happy, but won't improve the codebase in performance or readability (most likely it will worsen the readability !).

So adding a `#[allow(clippy::...)]` on a precise part of the code is the right approach here